src/Symfony/Component/Notifier/Message/SmsMessage.php

```diff
@@ -93,8 +95,18 @@ public function getTransport(): ?string
         return $this->transport;
     }
 
+    /**
+     * @return $this
+     */
+    public function options(?MessageOptionsInterface $options): self
```

Comment:
```suggestion
    public function options(MessageOptionsInterface $options): self
```
Setting null here makes no sense

Reply:
Fixed, I added the ability to set null because I thought maybe someone needs to reset options in some cases